client/my-sites/earn/memberships/add-edit-plan-modal.jsx

```diff
@@ -216,17 +216,17 @@ const RecurringPaymentsPlanAddEditModal = ( {
 			<>
 				<p>
 					{ product
-						? translate( 'Edit your existing Recurring Payments plan.' )
+						? translate( 'Edit your existing payment plan.' )
 						: translate(
-								'Each amount you add will create a separate Recurring Payments plan. You can create multiple plans.'
+								'Each amount you add will create a separate payment plan. You can create many of them.'
 						  ) }
 				</p>
 				<FormFieldset>
 					<FormLabel htmlFor="currency">{ translate( 'Select price' ) }</FormLabel>
 					{ product && (
 						<Notice
 							text={ translate(
-								'Updating the price will not affect existing subscribers, who will pay what they were originally charged.'
+								'Updating the price will not affect existing subscribers, who will pay what they were originally charged on renewal.'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.